drivers/volume/portworx/pure.go

```diff
@@ -0,0 +1,20 @@
+package portworx
+
+const (
+	// PureDriverName is the name of the portworx-pure driver implementation
+	PureDriverName = "pure"
+)
+
+// pure is essentially the same as the portworx volume driver, just different in name. This way,
+// we can have separate specs for pure volumes vs. normal portworx ones
+type pure struct {
+	portworx
+}
+
+func (p *pure) Init(sched, nodeDriver, token, storageProvisioner, csiGenericDriverConfigMap string) error {
```

Comment:
This is a very nice way to implement this! 👏

Reply:
Thank @nikolaypopov , he had the original idea :grin: I agree, made it muuuuch easier than I thought